core/templates/local_vector.h

```diff
@@ -104,6 +104,15 @@ class LocalVector {
 		return false;
 	}
 
+	_FORCE_INLINE_ bool erase_unordered(const T &p_val) {
```

Comment:
Do we want to provide a `from` argument here?

Reply:
We can add it if we find a need for it. `erase` doesn't have a `from` yet anyway.